setup.py

```diff
@@ -2,7 +2,7 @@
 
 setup(
     name="betterproto",
-    version="1.2.3",
+    version="1.3.0+clst",
```

Comment:
Shouldn't this be 1.2.3+clst ?

Reply:
I think a version bump makes sense, but maybe 1.2.4+clst since it's more a bug fix? Also I think we'll need to make additional fixes for other aspects of betterproto so we'll further deviate from their 1.2.3

Reply:
But I see where it's confusing. Will test if something like `1.2.3+clst1`, `1.2.3+clst2` works

Reply:
Yeah it's more clear to me if we have an additional tag off of the tag we are "branching" from

Reply:
Changed to `1.2.3+clst1`